astropy/units/decorators.py

```diff
@@ -204,7 +218,8 @@ def wrapper(*func_args, **func_kwargs):
                 #   were specified in the decorator/annotation, or whether a
                 #   single string (unit or physical type) or a Unit object was
                 #   specified
-                if isinstance(targets, str) or not isinstance(targets, Sequence):
+                if (isinstance(targets, str)
+                        or not isinstance(targets, Sequence)):
```

Comment:
Was it that bad before? :laughing:

Reply:
I am a line-wrapping chaos monkey! 🐒

Reply:
we recently went with the 100 long lines 🙈 😄

Reply:
Allowing it, but not prohibiting breaks to make it 80, I hope... (for my packages, I'm a stricter PEP8 adherent...)